src/main/java/com/sforce/cd/apexUnit/client/utils/ApexClassFetcherUtils.java

```diff
@@ -432,5 +433,24 @@ public static void logTheFetchedApexClasses(String[] classIds) {
 			}
 		}
 	}
+	
+	/*
+	 * Returns a string array of namespace and class/trigger name
+	 * 
+	 * @param strLine - String value of user provided <NAMESPACE>.<CLASS/TRIGGER>
+	 */
+	public static Map<String, String> parseNamespaceAndName(String strLine){
+		Map<String, String> namespaceAndName = new HashMap<String, String>();
+		String [] parsedName = strLine.split("\\.");
+		if(parsedName.length==1){
+			namespaceAndName.put("name", strLine);
+		}else{
+			namespaceAndName.put("namespace", parsedName[0]);
+			namespaceAndName.put("name", parsedName[1]);
+		}
```

Comment:
Yes @adarsh-ramakrishna-sfdc thats correct. Its good to validate the format of the name provided. But since the scope of the PR only ensures to look in the default namespace, I ll remove the code that parses the namespace as well. 

I guess I was combining the parsing logic as well in this PR.